tests/testthat/test-integration-logisticRegression.R

```diff
@@ -0,0 +1,67 @@
+context("Logistic Regression: helper functions testing ")
+
+config <- list(
+  `graph.resolution` = '1x',
+  `the.link` = 'logit',
+  `model.name` = 'Basic_Logistic_Regression',
+  `used.weights` = FALSE,
+  `X Vars` = c('gre', 'gpa', 'rank'),
+  `Y Var` = 'admit'
+)
+
```

Comment:
For the automatic creation of workflows, the config names should match the question names. I will handle taking care of the changes needed to make this happen, but wanted to give you the heads up.